aviary/subsystems/propulsion/engine_deck.py

```diff
@@ -962,19 +931,9 @@ def build_mission(self, num_nodes, aviary_inputs) -> om.Group:
                                              units=units[SHAFT_POWER_CORRECTED],
                                              desc='maximum corrected shaft power that can currently be produced')
 
-        else:
-            # If engine does not use thrust, a separate component for max thrust is not
-            # necessary.
-            # Add unscaled max thrust as output of interpolator, which will have a
-            # default value of zero at every flight condition
-            engine.add_output('thrust_net_max_unscaled',
-                              self.data[THRUST],
-                              units=units[THRUST],
-                              desc='Current max net thrust produced (unscaled)')
-
         # add created subsystems to engine_group
         outputs = []
-        if getattr(self, 'use_t4', False):
+        if self.use_t4:
```

Comment:
it might be safer to keep getattr, or can we guarantee that `use_t4` will always be added?

Reply:
I remember getattr was causing some issue but I went back and checked and it seems like it is ok now - reverted that change